velox/dwio/parquet/tests/writer/ParquetWriterTest.cpp

```diff
@@ -0,0 +1,147 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "velox/common/base/tests/GTestUtils.h"
+#include "velox/dwio/common/Reader.h"
+#include "velox/dwio/parquet/reader/ParquetReader.h"
+#include "velox/dwio/parquet/tests/ParquetReaderTestBase.h"
+#include "velox/dwio/parquet/writer/Writer.h"
+#include "velox/vector/tests/utils/VectorTestBase.h"
+
+using namespace facebook::velox;
+using namespace facebook::velox::common;
+using namespace facebook::velox::test;
+
+namespace facebook::velox::parquet {
+class ParquetWriterTest : public dwio::parquet::ParquetReaderTestBase,
```

Comment:
It's odd for ParquetWriterTest to extend `ParquetReaderTestBase`.

Reply:
Okay , I will create a new ParquetWriterTestBase class.

Reply:
@pdabre12 Meng created a ParquetWriterTestBase here https://github.com/facebookincubator/velox/pull/6608
We can probably rebase off that.

Reply:
Cool , works!